smartsim/_core/entrypoints/telemetrymonitor.py

```diff
@@ -460,6 +460,11 @@ def _to_completed(
 
         if hasattr(job.entity, "status_dir"):
             write_path = pathlib.Path(job.entity.status_dir)
+        else:
```

Comment:
I can move this change into my working telemetry monitor branch if necessary. I don't want to deal with the merge conflict headache... Let me know if you want me to add it after you remove it

Reply:
Oh, good point! I don't want to remove this change as it will result in a failing CI, but I also agree that dealing with the merge conflict in #460 is probably a lot to add to an already big PR. Let's prioritize #460 getting merged and bring this in after it.